api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleServiceTest.java

```diff
@@ -48,11 +50,30 @@ void shouldCreateNewPaymentCycle() {
     }
 
     @Test
-    void shouldCreateNewPaymentCycleForEligibleClaim() {
+    void shouldCreateNewPaymentCycleWithExpectedDueDateForEligibleClaimWithPregnancyVouchers() {
         LocalDate today = LocalDate.now();
-        Claim claim = aValidClaim();
+        Claim claim = aClaimWithExpectedDeliveryDate(today);
+        List<LocalDate> datesOfBirth = List.of(LocalDate.now(), LocalDate.now().minusDays(2));
+        PaymentCycleVoucherEntitlement entitlement = aPaymentCycleVoucherEntitlementWithPregnancyVouchers();
+
+        PaymentCycle result = paymentCycleService.createAndSavePaymentCycleForEligibleClaim(claim, today, entitlement, datesOfBirth);
+
+        verifyPaymentCycleSavedCorrectly(today, claim, result);
+        assertThat(result.getVoucherEntitlement()).isEqualTo(entitlement);
+        assertThat(result.getEligibilityStatus()).isEqualTo(EligibilityStatus.ELIGIBLE);
+        assertThat(result.getPaymentCycleStatus()).isEqualTo(PaymentCycleStatus.NEW);
+        assertThat(result.getChildrenDob()).isEqualTo(datesOfBirth);
+        assertThat(result.getExpectedDeliveryDate()).isEqualTo(today);
+        assertThat(result.getTotalEntitlementAmountInPence()).isEqualTo(entitlement.getTotalVoucherValueInPence());
+        assertThat(result.getTotalVouchers()).isEqualTo(entitlement.getTotalVoucherEntitlement());
+    }
+
+    @Test
+    void shouldCreateNewPaymentCycleWithoutExpectedDueDateForEligibleClaimWithoutPregnancyVouchers() {
+        LocalDate today = LocalDate.now();
+        Claim claim = aClaimWithExpectedDeliveryDate(today);
```

Comment:
I think its misleading to have a valid EDD on the Claim for this test, we either need to not have one, or have one which is a couple of years in the past, that is a more representative scenario.

Reply:
Set to six months in the past.

Reply:
It's still showing today...